mmv1/third_party/terraform/services/dataflow/resource_dataflow_flex_template_job_test.go.erb

```diff
@@ -1331,15 +1332,14 @@ resource "google_storage_bucket_object" "schema" {
 EOF
 }
 
-resource "google_kms_key_ring" "keyring" {
+data "google_kms_key_ring" "keyring" {
   name     = "%s"
   location = "global"
 }
```

Comment:
Can we remove the key ring from the test. This test is failing when I run it on our CI.

```
------- Stdout: -------
=== RUN   TestAccDataflowFlexTemplateJob_withKmsKey
=== PAUSE TestAccDataflowFlexTemplateJob_withKmsKey
=== CONT  TestAccDataflowFlexTemplateJob_withKmsKey
    vcr_utils.go:152: Step 1/2 error: Error running pre-apply refresh: exit status 1
        Error: projects/ci-test-project-188019/locations/global/keyRings/projects/ci-test-project-188019/locations/us-central1/keyRings/tftest-shared-keyring-1 not found
          with data.google_kms_key_ring.keyring,
          on terraform_plugin_test.tf line 33, in data "google_kms_key_ring" "keyring":
          33: data "google_kms_key_ring" "keyring" {
--- FAIL: TestAccDataflowFlexTemplateJob_withKmsKey (2.76s)
FAIL
```